frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -226,7 +226,7 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
         </Stack>
       </Form>
       {error && (
-        <Alert isInline variant="danger" title="Error creating model server">
+        <Alert className="pf-u-mt-lg" isInline variant="danger" title="Error creating model server">
```

Comment:
@DaoDaoNoCode @andrewballantyne I would like to know your opinion on this as you have more patternfly experience here, we try to avoid using custom classes, does that rule apply to patternfly classes too?
If so @pnaik1 we might wanna rework this solution trying to use some [layout](https://www.patternfly.org/v4/layouts/stack) strcuture.

Reply:
Correct, we should avoid using `className` or custom CSS if possible. Using the Stack layout component here would be a great solution. Another solution is you could wait until the feature freeze this Friday, at that time, the Explanability feature will be merged into main branch, where I created a [DashboardModalFooter](https://github.com/opendatahub-io/odh-dashboard/blob/f/mserving-metrics/frontend/src/concepts/dashboard/DashboardModalFooter.tsx) component, and you can pass it to the `footer` props in your Modal. So you don't need to worry about creating the layout by yourself.

Reply:
@DaoDaoNoCode is there any ETA for the [f/mserving-metrics](https://github.com/opendatahub-io/odh-dashboard/tree/f/mserving-metrics) to hit `main`? so we can wait for that or either use any other alternative

Reply:
@DaoDaoNoCode the [DashboardModalFooter](https://github.com/opendatahub-io/odh-dashboard/blob/f/mserving-metrics/frontend/src/concepts/dashboard/DashboardModalFooter.tsx) component is not yet merged to the main , so do I need to wait till it gets merged or proceed with the another solution?

Reply:
Yeah, it was last Friday, but it's a sprint delay, so it would be ~2.5 weeks later until it hits the main branch. You can copy-paste this file into the folder in this PR and update your modal to use it.

Reply:
yeah sure @DaoDaoNoCode

Reply:
I have made the required changes